<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7R54CV9" failures="1" tests="1" name="com.mystore.testcases.TC_MyAccountPageTest2" time="17.169" errors="0" timestamp="2022-10-01T17:52:26 IST" skipped="0">
  <testcase classname="com.mystore.testcases.TC_MyAccountPageTest2" name="verifysearchBox" time="17.169">
    <failure message="List is mistmatch expected [Summer Dresses &gt; Printed Summer Dress
T-shirts &gt; Faded Short Sleeve T-shirts
Blouses &gt; Blouse
Evening Dresses &gt; Printed Dress
Summer Dresses &gt; Printed Summer Dress
Summer Dresses &gt; Printed Chiffon Dress
Casual Dresses &gt; Printed Dress] but found [Summer Dresses &gt; Printed Summer Dress
Summer Dresses &gt; Printed Summer Dress
Summer Dresses &gt; Printed Chiffon Dress
T-shirts &gt; Faded Short Sleeve T-shirts
Blouses &gt; Blouse
Evening Dresses &gt; Printed Dress
Casual Dresses &gt; Printed Dress]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: List is mistmatch expected [Summer Dresses > Printed Summer Dress
T-shirts > Faded Short Sleeve T-shirts
Blouses > Blouse
Evening Dresses > Printed Dress
Summer Dresses > Printed Summer Dress
Summer Dresses > Printed Chiffon Dress
Casual Dresses > Printed Dress] but found [Summer Dresses > Printed Summer Dress
Summer Dresses > Printed Summer Dress
Summer Dresses > Printed Chiffon Dress
T-shirts > Faded Short Sleeve T-shirts
Blouses > Blouse
Evening Dresses > Printed Dress
Casual Dresses > Printed Dress]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at com.mystore.testcases.TC_MyAccountPageTest2.verifysearchBox(TC_MyAccountPageTest2.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifysearchBox -->
  <system-out/>
</testsuite> <!-- com.mystore.testcases.TC_MyAccountPageTest2 -->
